kdwsdl2cpp/src/converter_complextype.cpp

```diff
@@ -253,21 +253,24 @@ void Converter::convertComplexType(const XSD::ComplexType *type)
 // Called for each element and for each attribute of a complex type, as well as for the base class "value".
 QString Converter::generateMemberVariable(const QString &rawName, const QString &typeName, const QString &inputTypeName, KODE::Class &newClass, XSD::Attribute::AttributeUse use, bool usePointer, bool polymorphic)
 {
+    const bool optional = (use == XSD::Attribute::Optional);
+    const bool prohibited = (use == XSD::Attribute::Prohibited);
+    const KODE::Function::AccessSpecifier access = (prohibited) ? KODE::Function::Private : KODE::Function::Public;
+
     // member variable
     const QString storageType = usePointer ? pointerStorageType(typeName) : typeName;
     KODE::MemberVariable variable(rawName, storageType);
     addVariableInitializer(variable);
```

Comment:
This could move to the `else` branch of your new `if`.

It would make the code more readable: we either initialize one way or the other, not both.